tests/test_managers/test_inheritance_manager.py

```diff
@@ -115,6 +115,14 @@ def test_prior_select_related(self):
                 "inheritancemanagertestchild2").get(pk=self.child1.pk)
             obj.inheritancemanagertestchild1
 
+    def test_select_subclass_num_queries(self):
+        with self.assertNumQueries(1):
+            objs = self.get_manager().select_subclasses(
+                "inheritancemanagertestchild1",
+                "inheritancemanagertestchild2",
+            )
+            list(objs)
```

Comment:
If the test is conditional on django version, is there anything we want to assert on a django version basis here?

Reply:
@tony I guess the fact that tox is testing with all django and python supported version helps to make sure the test passes with all supported configuration.